src-built-in/components/linkerTester/linkerTester.js

```diff
@@ -0,0 +1,94 @@
+(() => {
+	"use strict";
+
+	document.addEventListener("DOMContentLoaded", () => {
+		// #region Elements
+		const dataType = document.getElementById("dataType");
+		const data = document.getElementById("data");
+		const sendButton = document.getElementById("send");
+		const received = document.getElementById("received");
+		// #endregion
+
+		let currentDataType = dataType.value;
+
+		/**
+		 * Function to handle data received over the linker.
+		 * 
+		 * @param {any} data 
+		 */
+		const printOutput = (title, data) => {
+			// Add received data to text field.
+			const titleDiv = document.createElement("div");
+			titleDiv.className = "title";
+			titleDiv.appendChild(document.createTextNode(title));
+
+			const dataBody = document.createElement("pre");
+			dataBody.innerHTML = data;
+
+			received.appendChild(titleDiv);
+			received.appendChild(dataBody);
+		};
+
+		FSBL.addEventListener("onReady", () => {
```

Comment:
For the sake of readability I'd request that you take these callbacks and put them into named functions. We're making a hard push for readability on the other side, so I think we should push for it here as well.